.github/workflows/release-please.yml

```diff
@@ -16,3 +16,6 @@ jobs:
           token: ${{ secrets.HCLOUD_BOT_TOKEN }}
           config-file: .github/release-please-config.json
           manifest-file: .github/release-please-manifest.json
+
+          extra-files: |
+            version/version.go
```

Comment:
This must go in the `.github/release-please-config.json` file now that we use the release-please action v4.

See https://github.com/hetznercloud/hcloud-python/blob/984022fd3888ef856be83de82554d55a8af18dba/.github/release-please-config.json#L10

Reply:
Oh right. The path is wrong anyway, it's missing `internal/`